app/analytics.go

```diff
@@ -244,96 +227,80 @@ func (a *App) GetAnalytics(name string, teamID string) (model.AnalyticsRows, *mo
 		rows[4] = &model.AnalyticsRow{Name: "command_count", Value: 0}
 		rows[5] = &model.AnalyticsRow{Name: "session_count", Value: 0}
 
-		iHookChan := make(chan store.StoreResult, 1)
-		go func() {
-			c, err2 := a.Srv().Store.Webhook().AnalyticsIncomingCount(teamID)
-			iHookChan <- store.StoreResult{Data: c, NErr: err2}
-			close(iHookChan)
-		}()
-
-		oHookChan := make(chan store.StoreResult, 1)
-		go func() {
-			c, err2 := a.Srv().Store.Webhook().AnalyticsOutgoingCount(teamID)
-			oHookChan <- store.StoreResult{Data: c, NErr: err2}
-			close(oHookChan)
-		}()
-
-		commandChan := make(chan store.StoreResult, 1)
-		go func() {
-			c, nErr := a.Srv().Store.Command().AnalyticsCommandCount(teamID)
-			commandChan <- store.StoreResult{Data: c, NErr: nErr}
-			close(commandChan)
-		}()
-
-		sessionChan := make(chan store.StoreResult, 1)
-		go func() {
-			count, err2 := a.Srv().Store.Session().AnalyticsSessionCount()
-			sessionChan <- store.StoreResult{Data: count, NErr: err2}
-			close(sessionChan)
-		}()
-
-		var fileChan chan store.StoreResult
-		var hashtagChan chan store.StoreResult
+		var g errgroup.Group
```

Comment:
I'd suggest naming this to `g2` or `gInternal` to avoid confusion.

Reply:
done